src/app/notebook/notebook-helpers.ts

```diff
@@ -0,0 +1,47 @@
+import { MimeBundle } from './notebook-types';
+
+// A list of the mime types we will display, ordered from "best" to "worst".
```

Comment:
(code) elaborate on what best means here, guessing best kind of image or something image like?

Reply:
Then again, why no svg love? Maybe this is dictated by the Python notebooks format?

Reply:
I wasn't sure how badly the sanitizers would mangle svg, so I left it off the list.  We could do some experiments and see if the common constructs pass through ok, but recommend that in another later ticket.